Makefile

```diff
@@ -1,14 +1,12 @@
-.PHONY: all
-all:
-	@echo "This make file is intended for releasing only. In order to build any of Tracee's components, use their respective make files."
-
-# RELEASE_TAG must be set to specify a name for the release
 OUT_DIR ?= dist
 OUT_DOCKER ?= tracee
 OUT_ARCHIVE := $(OUT_DIR)/tracee.tar.gz
 OUT_CHECKSUMS := $(OUT_DIR)/checksums.txt
 PUSH_DOCKER_REPO ?= aquasec/tracee
 PUSH_DOCKER_TAG ?= $(RELEASE_TAG:v%=%)
+RELEASE_FILES := LICENSE $(OUT_DIR)/tracee-ebpf $(OUT_DIR)/tracee-rules $(OUT_DIR)/rules
+# RELEASE_TAG must be set for the release target
+VERSION ?= $(if $(RELEASE_TAG),$(RELEASE_TAG),$(shell $(CMD_GIT) describe --tags 2>/dev/null || echo '0'))
```

Comment:
Can we now remove these same lines from tracee-ebpf/Makefile and other similar stuff as well?

Reply:
no because it's used to determine the name of the bpf obj (tracee.bpf.linuxversion.traceeversion.o)

Reply:
but we could clean up the tracee-ebpf makefile depending on what we decide to do with #255 (I'm leaving it to that PR)